new blogpost - ChatGPT Sentinel blogpost (eng)

BenAarvik opened the pull request:
#57

BenAarvik committed:
add blogpost

Signed-off-by: Bendik Aarvik <38646970+BenAarvik@users.noreply.github.com>

BenAarvik committed:
add tags

Signed-off-by: Bendik Aarvik <38646970+BenAarvik@users.noreply.github.com>

BenAarvik committed:
add featured image

Signed-off-by: Bendik Aarvik <38646970+BenAarvik@users.noreply.github.com>

remiks commented:
Good article. Some feedback:

1. Description needs to be a bit longer. As it is shown in the frontpage of the blog. Think ingress style.
2. Use correct md syntax. `## **Is not correct**` `## This is correct`
3. Use only lowercase in slug `slug: streamlining-incident-Management-with-chatgpt
4. Dont bullet indent descriptions, does not make any sense. (Line 45 +). Better to use h3 here.
5. Next time, do a spell check in word or Chatgpt ;) before submitting, was _a lot_ of spelling issues.

remiks committed:
Update index.mdx

Good article. Some feedback:

1. Description needs to be a bit longer. As it is shown in the frontpage of the blog. Think ingress style.
2. Use correct md syntax. `## **Is not correct**` `## This is correct`
3. Use only lowercase in slug `slug: streamlining-incident-Management-with-chatgpt
4. Dont bullet indent descriptions, does not make any sense. (Line 45 +). Better to use h3 here.
5. Next time, do a spell check in word or Chatgpt ;) before submitting, was _a lot_ of spelling issues.

Commit:
Merge branch 'main' into main

Commit:
Merge branch 'main' into main
